### PR TITLE
created two files one for modules and one for  class

### PR DESCRIPTION
**Pull request by businesskaniki**
we created a module and imported the  module in our enumerable class

### CODE REVIEW

**Review by mmsesay — APPROVED** [review]
Comments Below

File: enums.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+list = MyList.new([1, 2, 3, 4])
+# Test #all?
+
+puts(list.all? { |e| e < 5 })
+puts(list.all? { |e| e > 5 })
+# Test #any?
+puts(list.any? { |e| e == 2 })
+puts(list.any? { |e| e == 5 })
+# Test #filter?
+print(list.filter?(&:even?))
```
Comment (mmsesay) [inline enums.rb:23]: - [OPTIONAL] Please remove these print statements. They will cause some call errors when testing your project. You can just use them in the ruby console/shell to evaluate if your project gives the expected outputs.

**Review by mmsesay — APPROVED** [review]
Hi @businesskaniki & @tarek102, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
 - Good Readme :heavy_check_mark: 
 - No linter error :heavy_check_mark: 
 - Good PR title :heavy_check_mark: 
 - Good PR summary :heavy_check_mark:  

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: enums.rb (hunk 1 above)
Comment (businesskaniki) [reply enums.rb:23]: okay thnkyou

**Pull request merged**